test/unit/base_test.rb

```diff
@@ -127,51 +127,63 @@ class Aggregate::BaseTest < ActiveSupport::TestCase
       assert_nil @agg.new.respond_to_without_attributes?("could", "be", "anything")
     end
 
-    should "provide comparable methods for instances" do
-      @first  = @agg.new(name: "Bob", address: "1812 clearview", zip: 93_101)
-      @same   = @agg.new(name: "Bob", address: "1812 clearview", zip: 93_101)
-      @second = @agg.new(name: "Bob", address: "1812 clearview", zip: 93_102)
-
-      assert @first == @same
-      assert @first != @second
-      assert @first < @second
-    end
+    context "comparisons" do
+      should "provide comparable methods for instances" do
```

Comment:
Nit: You didn't write this test originally but this test and the one below are using instance variables when they aren't needed.